pegasus/sites.v3/hourofcode.com/views/census_questions_followup.haml

```diff
@@ -0,0 +1,65 @@
+:ruby
+  topic_options = [
```

Comment:
I wonder if there's also an array & loop abbreviation for this code, too.

Reply:
Tell me more... This is an array of `topic_options` that I loop through on line 47.